packages/qwik-query/src/index.ts

```diff
@@ -0,0 +1,16 @@
+// Re-export core
+export * from '@tanstack/query-core'
+
+export * as QueryClientProvider from './QueryClientProvider'
+
+export { useQuery } from './useQuery'
+export { useQueries } from './useQueries'
+
+export { useInfiniteQuery } from './useInfiniteQuery'
+export { useIsFetching } from './useIsFetching'
+
+export { useMutation } from './useMutation'
+export { useIsMutating } from './useIsMutating'
```

Comment:
v5 has a new `useMutationState` hook that you might want to implement in qwik as well